.circleci/real_config.yml

```diff
@@ -4106,7 +4126,6 @@ workflows:
           name: build-react-oss
           dev-mode: true
       - test-e2e-react:
-          name: test-e2e-react-oss
```

Comment:
I now have https://www.youtube.com/watch?v=na47wMFfQCo stuck in my head. "I've been through CI in a test with no name."

Reply:
oh actually i think i deleted the wrong line here. good catch